skyportal/models/obj.py

```diff
@@ -347,114 +346,6 @@ class Obj(Base, conesearch_alchemy.Point):
         doc="Notifications regarding the object sent out by users",
     )
 
-    @hybrid_method
-    def last_detected_at(self, user):
-        """UTC ISO date at which the object was last detected above a given S/N (3.0 by default)."""
-        detections = [
-            phot.iso
-            for phot in Photometry.query_records_accessible_by(user)
-            .filter(Photometry.obj_id == self.id)
-            .all()
-            if phot.snr is not None and phot.snr > PHOT_DETECTION_THRESHOLD
-        ]
-        return max(detections) if detections else None
-
```

Comment:
I'm not sure we want to get rid of these. They include the detection stats with permissions, where PhotStats are not.

Reply:
I debated this for a bit, but I actually think it's pretty confusing to have these too if the rest of the app will use the other table (even if it is arguably more exact).

Reply:
ok